### PR TITLE
Handle race condition with polling

### LINKED ISSUE #492: Error Cannot assign work to worker in state 'inactive'
https://sentry.io/rainforest-1/tester-chrome-extension/issues/157689482/

```
Error: Cannot assign work to worker in state 'inactive'
  at reducer (src/reducers/worker.js:36:16)
  at r (node_modules/redux-actions/lib/handleAction.js:31:33)
  at reduce (node_modules/reduce-reducers/lib/index.js:13:13)
  at reducer (node_modules/reduce-reducers/lib/index.js:12:20)
  at reducer (node_modules/redux-actions/lib/handleActions.js:33:11)
...
(3 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by shosti**
Fixes #492

@rainforestapp/tester-product @ukd1

### CODE REVIEW

File: lib/handlePolling.js [hunk 1]
```diff
@@ -31,7 +31,16 @@ var handlePolling = function handlePolling(store) {
         throw new Error(msg);
       }
 
-      store.dispatch((0, _actions.assignWork)({ url: data.url }));
+      // Double-check the worker state in case something changed since the fetch
+      // started.
+
+      var _store$getState = store.getState();
```
Comment (curtis-rainforestqa) [inline lib/handlePolling.js]: is this var necessary if we only use it in one spot?

Comment (curtis-rainforestqa) [inline lib/handlePolling.js]: nvm, it's better for readability/clarity this way

**Comment by curtis-rainforestqa** [comment]
lgtm, 👍

File: lib/handlePolling.js (hunk 1 above)
Comment (shosti) [inline lib/handlePolling.js]: btw that's actually the "built" code, real code is all under src.

**Pull request merged**